src/components/EntryList/Entry-list.css

```diff
@@ -0,0 +1,16 @@
+.entry-list {
+  border: 1px solid black;
+  margin: 5px;
+  height: auto;
+  max-height: 25vh;
+  max-width: 25vw;
+  overflow-wrap: break-word;
+  padding: 15px;
+  inline-size: 150px;
```

Comment:
[Inline size](https://css-tricks.com/almanac/properties/i/inline-size/) is generally used when you're expecting the orientation of the text to change. Width would serve your purpose just fine here